src/migrations/migrations-truffle-5/4_deploy_Pool.js

```diff
@@ -41,17 +41,11 @@ async function migrate({
     const contractsMapped = contractArtFilePaths.map(path => TC(require(path)))
     // Set deployer provider to each contract
     contractsMapped.forEach(tcArt => tcArt.setProvider(deployer.provider));
-    ([dxProxy, etherToken, tokenB] = await Promise.all(contractsMapped.map(contract => process.env.TOKEN_B_ADDRESS ? contract.at(process.env.TOKEN_B_ADDRESS) : contract.deployed())))
+    ([dxProxy, etherToken, otherToken] = await Promise.all(contractsMapped.map((contract, idx) => (process.env.TOKEN_B_ADDRESS && idx == 2) ? contract.at(process.env.TOKEN_B_ADDRESS) : contract.deployed())))
```

Comment:
ubernit: might be worth to put this in a { block } since it's a relatively tough to parse one line.